### PR TITLE
change python3 version in bazel containers

### PR DESCRIPTION
**Pull request by nlopezgi**
* The py3 version in rbe containers is problematic for add-apt-repo commands
* For the bazel containers that dont require as an up to date version of py3 use apt-get to install that.

### CODE REVIEW

File: container/common/python3.yaml [hunk 1]
```diff
@@ -15,4 +15,4 @@ commandTests:
 - name: 'python3-version'
```
Comment (xingao267) [inline container/common/python3.yaml]: We can add a new command test to test add_apt_repository

File: container/debian8/builders/rbe-debian8/BUILD [hunk 2]
```diff
@@ -65,7 +65,7 @@ toolchain_container(
         "//container/debian8/layers/go:go-ltl",
         "//container/debian8/layers/java:java-ltl",
         "//container/debian8/layers/java:java10-ltl",
-        "//container/debian8/layers/python:python-ltl",
+        "//container/debian8/layers/python:python-rbe-ltl",
```
Comment (xingao267) [inline container/debian8/builders/rbe-debian8/BUILD]: You probably also need to update the internal container reproduction test after this change.

Comment (nlopezgi) [reply container/debian8/builders/rbe-debian8/BUILD]: thanks for the reminder

File: container/common/python3.yaml (hunk 1 above)
Comment (nlopezgi) [reply container/common/python3.yaml]: done

**Pull request merged**